app/Core/Database.php

```diff
@@ -1,6 +1,7 @@
 <?php
 namespace Core;
-
+$config_data = parse_ini_file("dbconfig.ini");
```

Comment:
This shouldn't be in Core. It should be in the root directory (the some place where the index is).

Reply:
The ini file itself?